### ISSUE TITLE
[MAS Intake] <REQUEST TITLE>

### ISSUE THREAD

**Issue by ukalaylar** [Shopify/mas-intake#190]
### Your Name

Kayla

### Your Team

MAS

### TL;DR of the request

test for workflow automations

### Urgency of the request

Not Urgent / 2 weeks or more

### Request Category

Other

### If 'Consultation' was chosen above, select the appropriate service

None

### If 'Advisor/Specialist Request' was chosen above, provide the appropriate details, otherwise leave empty

1. Which of the following options best reflects your request? [FEEDBACK(Round table, Surveys) / COVERAGE(Events,Project time,Beta) / SECONDMENT / OTHER]
2. How much time and coverage is needed? [TIME ASK / DATE RANGE]
3. How many Advisors/Specialists are required? [PROVIDE A NUMBER]
4. Specify the name, the team and the region of the required individuals: [FULL NAME + TEAM CODE + REGION / UNSURE + THE REGION REQUIRED]
5. Please enter the name of the Senior Lead approving this request: [FULL NAME]
6. Is travel required? [YES / NO]
7. If travel is required, please describe who will deal with the logistics of the travel and which team will cover the costs: [LOGISTICS DETAILS] 
8. Has WFP been contacted? [YES / NO]

### Summary of the request

test

### What do you need from MAS?

test

### Date required by

10/10/2023
